apps/src/code-studio/components/aiTutor/warningModal.jsx

```diff
@@ -0,0 +1,53 @@
+import React, {useState} from 'react';
+import style from '@cdo/apps/aichat/views/chatWarningModal.module.scss';
+import {
+  BodyTwoText,
+  Heading2,
+  StrongText,
+} from '@cdo/apps/componentLibrary/typography';
+import AccessibleDialog from '@cdo/apps/templates/AccessibleDialog';
+import Button from '@cdo/apps/templates/Button';
+
+const WarningModal = () => {
+  const [open, setOpen] = useState(true);
+  console.log(open);
```

Comment:
you can remove this debugging console.log

Reply:
Removed!